modules/tensor_mechanics/src/materials/ComputeCosseratElasticityTensor.C

```diff
@@ -31,8 +31,8 @@ ComputeCosseratElasticityTensor::ComputeCosseratElasticityTensor(const InputPara
     _elastic_flexural_rigidity_tensor(
         declareProperty<RankFourTensor>("elastic_flexural_rigidity_tensor"))
 {
-  // all tensors created by this class are always constant in time
-  issueGuarantee(_elasticity_tensor_name, Guarantee::CONSTANT_IN_TIME);
+  if (!isParamValid("elasticity_tensor_prefactor"))
```

Comment:
I'm always a little leery of `isParamValid()`. Maybe it works fine like this, but it might be better to just do `if (_prefactor_function)`.

Reply:
There's nothing wrong with that method. Maybe you don't like the style but it's reliable.

Reply:
i think the ```isParamValid``` is a little more readable and therefore less error prone(?)  i'm a bit leery of it too - maybe it was buggy in the past or something - but it does seem to work here.   Anyway, overll i'm so glad that no Apps failed because of this!

Reply:
I've tried to use it a number of times and it hasn't behaved as I expected. If you've verified that it works correctly here, go for it.